openupgrade_scripts/apriori.py

```diff
@@ -14,6 +14,8 @@
     "account_commission": "account_commission_oca",
     "commission": "commission_oca",
     "sale_commission": "sale_commission_oca",
+    # OCA/e-commerce
+    "website_sale_product_attribute_filter_collapse": "website_sale",
```

Comment:
you have to put in the `merged_modules` section.

Reply:
Sorry for the mistake.